CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Note: Later matches take precedence
+
+# default owner
+*                         @DataDog/apm-go
+
+# tracing
+/contrib                  @DataDog/tracing-go
```

Comment:
I have created this team as a sub-team of apm-go and added myself, @knusbaum and @dianashevchenko as members.